storage/innobase/xtrabackup/test/t/pxb-2710-predict_memory.sh

```diff
@@ -6,8 +6,13 @@ require_debug_pxb_version
 MYSQLD_EXTRA_MY_CNF_OPTS="
 innodb-flush-log-at-trx-commit=0
 sync-binlog=0
-innodb-redo-log-capacity=512M
 "
+if is_server_version_higher_than 8.0.29
+then
+  MYSQLD_EXTRA_MY_CNF_OPTS="${MYSQLD_EXTRA_MY_CNF_OPTS:-""}
+  innodb-redo-log-capacity=512M
+  "
+fi
```

Comment:
else 
innodb-log-file-size=512M?

Reply:
it's not needed for pre 8.0.30 as we do not remove the files (wrap around) unless we actually write more than 48M*2